WooCommerce/Classes/System/WooConstants.swift

```diff
@@ -328,6 +328,12 @@ extension WooConstants {
         ///
         case dhlExpressInstructions = "https://www.dhl.com/global-en/home/our-divisions/freight/customer-service/dangerous-goods-and-prohibited-items.html"
 
+        case subscriptionsExtensionURL = "https://woocommerce.com/products/woocommerce-subscriptions/"
+
+        case productBundlesExtensionUrl = "https://woocommerce.com/products/product-bundles/"
+
+        case compositeProductsExtensionUrl = "https://woocommerce.com/products/composite-products/"
```

Comment:
super nit: URL postfix probably isn't necessary because the enum already indicates it's a URL?
```suggestion
        case subscriptionsExtension = "https://woocommerce.com/products/woocommerce-subscriptions/"

        case productBundlesExtension = "https://woocommerce.com/products/product-bundles/"

        case compositeProductsExtension = "https://woocommerce.com/products/composite-products/"
```